src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileTabCharacter/MoveCodeInSameFileWithMinorChanges/newline/expected.txt

```diff
@@ -0,0 +1,3 @@
+Starting audit...
```

Comment:
you don't need `expected.txt` files here, because you already created UTs and defined what is expected there

Reply:
done